plugins/module_utils/network/sonic/config/lldp_interfaces/lldp_interfaces.py

```diff
@@ -80,6 +82,24 @@ def get_lldp_interfaces_facts(self):
             return []
         return lldp_interfaces_facts
 
+    def range_to_list(self, vrange):
+        """ Convert a range of values to a list
+        :rtype: A list
+        :returns: The list of values
+        """
+        range_list = []
+        vlist = vrange.split(",")
+        for v in vlist:
+            if isinstance(v, int):
+                range_list.append(v)
+            elif "-" in v:
+                ranges = v.split("-")
```

Comment:
Why is this being done instead of just changing the "-" to ".."? Please change this conversion to use the SONIC range format instead of converting the range to a list of individual vlans or explain why that won't work.

Reply:
modified the function to send the values as such instead of expanding them

Reply:
I am marking this "Issue" as "Resolved" because of the changes that have been made, but have added a comment on the use of an exception instead of a conditional branch.